peachjam/js/components/FilterFacets/index.vue

```diff
@@ -104,19 +65,43 @@ export default {
       }
       return data;
     },
+
     clearFacet (fieldName) {
+      const queryString = window.location.search;
```

Comment:
for `clearFacet` and `clearAll` we dont need update `window.location.href`.  Let the form submission in `ListFacets.vue` handle that. So lets update `clearFacet` and `clearAll` to this
```
clearFacet (fieldName) {
      const data = this.clearSingleFacet(this.modelValue, fieldName);
      this.$emit('update:modelValue', [...data]);
    },

    clearAll () {
      let data = this.modelValue;
      this.modelValue.forEach((facet) => {
        data = this.clearSingleFacet(data, facet.name);
      });
      this.$emit('update:modelValue', [...data]);
    },
```